src/test/java/com/iexec/common/utils/IexecEnvUtilsTest.java

```diff
@@ -83,7 +83,7 @@ void getComputeStageEnvList() {
                         "IEXEC_IN=/iexec_in",
                         "IEXEC_OUT=/iexec_out",
                         "IEXEC_DATASET_ADDRESS=datasetAddress",
```

Comment:
```suggestion
```
If we remove this from `IexecEnvUtils#getComputeStageEnvMap`, we have to remove it here too.